swabbie-edda/src/main/kotlin/com/netflix/spinnaker/swabbie/edda/EddaService.kt

```diff
@@ -43,4 +46,22 @@ interface EddaService {
 
   @GET("/api/v2/aws/autoScalingGroups;_expand")
   fun getAutoScalingGroups(): List<AmazonAutoScalingGroup>
+
+  @GET("/api/v2/aws/images;_expand:(imageId,name,description,state,tags)")
```

Comment:
This whole class works with clouddriver config, right? So that if you don't have edda and are hitting aws directly this does the right thing?

Reply:
Edda is set by default as of now but that will change when we add aws support.
At that point, we should be able to toggle the edda flag and be able to talk to amazon directly.